R/pkg_ref_cache_repo_pkg_website.R

```diff
@@ -0,0 +1,37 @@
+#' Cache package's Website URL
+#'
+#' @inheritParams pkg_ref_cache
+#' @family package reference cache
+#'
+pkg_ref_cache.pkg_website <- function(x, name, ...) {
+  UseMethod("pkg_ref_cache.pkg_website")
+}
+
+pkg_ref_cache.pkg_website.pkg_remote <- function(x, name, ...) {
+
+  db  <- rvest::html_table(x$web_html)[[1]]
+  url <- db[grep("URL",db[,1], ignore.case = TRUE) ,2]
+  if(length(url) == 0) url <- NA
+  url
+}
+
+pkg_ref_cache.pkg_website.pkg_install <- function(x, name, ...) {
+
+  get_description_field(x$description, "url")
+
+}
+
+pkg_ref_cache.pkg_website.pkg_source <- function(x, name, ...) {
+
+  get_description_field(x$description, "url")
+
+}
```

Comment:
On second though, I think we should avoid `.default` so that we get appropriate errors when methods aren't defined. Instead, I'd suggest you do

```suggestion
pkg_ref_cache.pkg_website.pkg_install <- function(x, name, ...) {
  get_description_field(x$description, "url")
}

pkg_ref_cache.pkg_website.pkg_source <- pkg_ref_cache.pkg_website.pkg_install
```